starboard/android/apk/app/src/main/java/dev/cobalt/media/AudioTrackBridge.java

```diff
@@ -53,6 +53,9 @@ private static int getBytesPerSample(int audioFormat) {
         return 2;
       case AudioFormat.ENCODING_PCM_FLOAT:
         return 4;
+      case AudioFormat.ENCODING_AC3:
```

Comment:
Consider to use `AudioFormat.getFrameSizeInBytes()`.

Reply:
Fixed it as getBufferSizeInFrames() returns the size of the AudioTrack buffer in bytes if it is compressed data.